rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
This rule is perhaps closer to what you are looking for:

```
SUBSYSTEM=="input", ENV{ID_INPUT_ACCELEROMETER}=="1", SUBSYSTEMS=="acpi", \
  IMPORT{builtin}="hwdb 'sensor:modalias:acpi:$attr{hid}:$attr{[dmi/id]modalias}'", \
  GOTO="sensor_end"
```

It works and applies the matrix to both input and event devices.

It relies on the fact that input and event devices don't provide a hid attribute, so it always fall back on the matched ACPI parent for it. The rule is now ACPI-specific as a result. What do you think?